### DISCUSSION TITLE
Arduino IDE 2.3.3. repeats downloading ESP Async Webserver@3.3.20

### DISCUSSION THREAD

**Discussion by PaulskPt** [mathieucarbou/ESPAsyncWebServer#145] [General]
On my MS Windows 11 Pro desktop PC I have installed Arduino IDE v2.3.3. Every time when I start the IDE it tries to download and install ESP Async WebServer@3.3.20.
This is just to inform anybody who is responsible for updating this library.

**Comment by mathieucarbou** [comment]
Hello @PaulskPt : this has nothing to do with the project, but has to do with you using the IDE ;-)
At startup, Arduino IDE will try to update its already downloaded libraries.

**Reply by PaulskPt** [reply]
Thank you for your reply!
